examples/bridge-app/linux/main.cpp

```diff
@@ -358,6 +402,33 @@ void HandleDeviceSwitchStatusChanged(DeviceSwitch * dev, DeviceSwitch::Changed_t
     }
 }
 
+void HandleDevicePowerSourceStatusChanged(DevicePowerSource * dev, DevicePowerSource::Changed_t itemChangedMask)
+{
+    if (itemChangedMask &
+        (DevicePowerSource::kChanged_Reachable | DevicePowerSource::kChanged_Name | DevicePowerSource::kChanged_Location))
+    {
+        HandleDeviceStatusChanged(static_cast<Device *>(dev), (Device::Changed_t) itemChangedMask);
+    }
+
+    if (itemChangedMask & DevicePowerSource::kChanged_BatLevel)
+    {
+        uint8_t batChargeLevel = dev->GetBatChargeLevel();
+        MatterReportingAttributeChangeCallback(dev->GetEndpointId(), ZCL_POWER_SOURCE_CLUSTER_ID,
+                                               ZCL_POWER_SOURCE_BAT_CHARGE_LEVEL_ATTRIBUTE_ID, CLUSTER_MASK_SERVER,
```

Comment:
Please use the namespaced cluster/attribute ids.

Reply:
Sure.